hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -271,14 +256,38 @@
                   ]
                 }
     },
-# outputs
-    { multireg: { name:     "MIO_OUTSEL",
-                  desc:     "Mux select for MIO outputs.",
+
+//////////////////////////
+// MIO Outputs          //
+//////////////////////////
+    { multireg: { name:     "MIO_OUTSEL_REGWEN",
+                  desc:     "Register write enable for MIO output selects.",
                   count:    "NMioPads",
-                  swaccess: "rw",
-                  hwaccess: "hro",
-                  regwen:   "REGEN",
-                  cname:    "OUT",
+                  swaccess: "rw0c",
+                  hwaccess: "none",
+                  cname:    "MIO_OUTSEL",
+                  fields: [
+                    { bits:   "0",
+                      name:   "EN",
+                      desc:   '''
+                              Register write enable bit.
+                              If this is cleared to 0, the corresponding MIO_OUTSEL
+                              is not writable anymore.
+                              ''',
+                      resval: "1",
+                    }
+                  ]
+                }
+    },
+    { multireg: { name:         "MIO_OUTSEL",
+                  desc:         "Mux select for MIO outputs.",
```

Comment:
could we change the description to something like..."for each muxable pad, select peripheral..."? It might just be me, but I always get kind of confused by just this direction, it makes me wonder whether it's pad->mio->peripheral (output from pinmux) or peripheral->mio->pad (ouput to pads)

Reply:
yeah this direction is confusing, let me change the description.